uber/config.py

```diff
@@ -377,4 +373,12 @@ def _is_intstr(s):
 
 c.SAME_NUMBER_REPEATED = r'^(\d)\1+$'
 
```

Comment:
I made the volunteer checklist configurable.  You'll see what this config looks like below, but this lets us do things like have different events use different steps on the checklist.  We'll want the hotel stuff as part of the checklist, but other events won't, and it's not there by default anymore.